pkgs/development/python-modules/py_stringmatching/default.nix

```diff
@@ -0,0 +1,29 @@
+
+{ lib
+, buildPythonPackage
+, numpy
+, six
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "py_stringmatching";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rjsx7iipn6svki21lmsza7b0dz9vkgmix696zryiv7gkhblqyb4";
+  };
+
+  # No tests in archive
+  doCheck = false;
+   
+  propagatedBuildInputs = [ numpy six ];
+
+  meta = with lib; {
+    description = "This project seeks to build a Python software package that consists of a comprehensive and scalable set of string tokenizers (such as alphabetical tokenizers, whitespace tokenizers) and string similarity measures (such as edit distance, Jaccard, TF/IDF).";
```

Comment:
Too long. This could be a `longDescription`. Also please split it into multiple lines.